docs/UserGuide.md

```diff
@@ -44,36 +44,28 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
 
 * Items in square brackets are optional.<br>
   e.g `n/NAME [r/SCHOOL RESIDENCE]` can be used as `n/John Doe r/RC4` or as `n/John Doe`.
-  
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
   
+* For commands with prefixs, 
```

Comment:
This is a typo, it should be "Prefixes" :)

Reply:
Ohhhh, yes, thanks for pointing it out!